fixtures/dom/src/components/fixtures/home.js

```diff
@@ -0,0 +1,90 @@
+const React = window.React;
+
+export default function Home() {
+  return (
+    <main>
+      <h1>DOM Test Fixtures</h1>
+      <p>
+        Use this site to test browser quirks and other behavior that can not be
+        captured through unit tests.
+      </p>
+      <section>
+        <h2>Tested Browsers</h2>
+        <table>
+          <thead>
+            <tr>
+              <th>Browser</th>
+              <th>Versions</th>
+              <th>Why/Notes</th>
```

Comment:
Minor UX nit: this "Why/Notes" column is mostly empty. The few notes we have would likely be better as footnotes